src/main/java/com/chat/yourway/repository/TopicRepository.java

```diff
@@ -35,4 +35,14 @@ WHERE to_tsvector('english', t.topic_name) @@ to_tsquery('english', :query)
   List<Topic> findAllFavouriteTopicsByContactEmail(String contactEmail);
 
   boolean existsByIdAndIsPublic(int topicId, boolean isPublic);
+
+  @Query(nativeQuery = true, value =
```

Comment:
Why are you using native queries and not Hibernate (HQL)?

Reply:
I had some problem with JPQL and decided to use native query because it's more easier implement for me